pkg/cmd/readyz/readyz.go

```diff
@@ -178,12 +180,21 @@ func (r *readyzOpts) Run() error {
 		return err
 	}
 
+	var tlsMinVersion uint16
+	switch tlsutil.TLSVersion(r.tlsMinVersion) {
+	case tlsutil.TLSVersion13:
+		tlsMinVersion = tls.VersionTLS13
+	case tlsutil.TLSVersion12:
+		tlsMinVersion = tls.VersionTLS12
+	}
```

Comment:
"go.etcd.io/etcd/client/pkg/v3/tlsutil" to the rescue :)

Reply:
https://github.com/etcd-io/etcd/blob/19aa0dbe8fd6317a237bae9b6ea52a4f1b445b19/client/pkg/tlsutil/versions.go#L32-L47

to be precise

Reply:
Got it this time :-)